cmd/kubebuilder/util/util.go

```diff
@@ -180,3 +180,18 @@ func IsNewVersion() bool {
 func ProjectExist() bool {
 	return IsNewVersion()
 }
+
+func IsProjectNotInitialized() bool {
+	dirs := []string{
+		"cmd",
+		"hack",
```

Comment:
I'm confused what is the purpose of this function.
`hack` only exists in an old kb project.

Reply:
The function checks if there exists an initialized project inside one directory.
This function is used when adding the default commands to `kubebuilder`. We want the default to be v1 commands, even for an empty folder.
There is a `hack` package in the new KB project. 
```
tree hack
hack
└── boilerplate.go.txt
```